### PR TITLE
Update authentication-aad-configure.md

### PR DESCRIPTION
**Pull request by JFolberth**
This pull request includes a small change to the `azure-sql/database/authentication-aad-configure.md` file. The change advises associating the Microsoft Entra admin ID with an Entra Group for better management.

* [`azure-sql/database/authentication-aad-configure.md`](diffhunk://#diff-ea96a7646503bd8e08fec3959d7ae89454311fe65708aff399818d979324039eR83-R84): Added a recommendation to tie the Microsoft Entra admin ID to an Entra Group, specifying that the Client ID of the Entra Group will be inserted into the `sys.database_principals` in the master table.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@nofield 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/authentication-aad-configure.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ For more information, see:
 
 Each [logical server](logical-servers.md) in Azure (which hosts SQL Database or Azure Synapse) starts with a single server administrator account that is the administrator of the entire server. Create a second administrator account as a Microsoft Entra account. This principal is created as a contained database user in the `master` database of the server. Administrator accounts are members of the **db_owner** role in every user database, and each user database is entered as the **dbo** user. For more information about administrator accounts, see [Managing Databases and Logins](logins-create-manage.md).
 
+It is advised that the ID associated as the Microsoft Entra admin is tied to an Entra Group. This Client ID of the Entra Group will be inserted into the `sys.database_principals` in the master table.
```
Comment (rwestMSFT) [inline azure-sql/database/authentication-aad-configure.md]:
```suggestion
You should associate the Microsoft Entra ID admin with a Microsoft Entra Group. This client ID of the Microsoft Entra Group is inserted into the `sys.database_principals` table in the `master` database. 
```

**Comment by WilliamDAssafMSFT** [comment]
@JFolberth Thanks for taking the time to submit this PR. This may be good guidance to make here, running it past some engineers to confirm if this isn't a little more complicated. Will update the PR accordingly.

**Comment by nofield** [comment]
Thanks @JFolberth for your suggestion here. This and other feedback has been added to the Entra docs, please see here: https://learn.microsoft.com/en-us/azure/azure-sql/database/authentication-aad-overview?view=azuresql#microsoft-entra-administrator

**Comment by rwestMSFT** [comment]
Marking this as resolved.

#please-close

**Pull request closed without merging**